pkg/riff/commands/application_create.go

```diff
@@ -142,6 +146,22 @@ func (opts *ApplicationCreateOptions) Exec(ctx context.Context, c *cli.Config) e
 		return err
 	}
 	c.Successf("Created application %q\n", application.Name)
+	if opts.Tail {
+		// cancel ctx when application becomes ready
+		ctx, cancel := context.WithCancel(ctx)
+		go func() {
+			defer cancel()
+			applicationWatch, err := c.Build().Applications(opts.Namespace).Watch(metav1.ListOptions{
+				ResourceVersion: application.ResourceVersion,
+			})
+			if err != nil {
+				return
+			}
+			defer applicationWatch.Stop()
+			k8s.WaitUntilReady(applicationWatch)
+		}()
+		return c.Kail.ApplicationLogs(ctx, application, time.Minute, c.Stdout)
```

Comment:
The `time.Minute` constant is repeated in the four similar blocks and should probably be factored out to preserve consistency.

Reply:
exposing since as a flag, with a default of 1 second.

Reply:
Er, the code is unchanged and still refers to `time.Minute`. What am I missing?

Reply:
actually done now